### PR TITLE
Rna strandness fix

### PR DESCRIPTION
**Pull request by qinqian**

### CODE REVIEW

File: RNA_pipeline/RNA_pipeline.wdl [hunk 1]
```diff
@@ -16,6 +19,9 @@ workflow RNA_pipeline {
   File input_bam_cram
   File reference_fasta
 
+  String is_stranded = "true"
+  String is_stranded_rnaseqc = "RF"
```
Comment (5im1z) [inline RNA_pipeline/RNA_pipeline.wdl:23]: not sure if it's a good idea to hardcode these

File: RNA_pipeline/RNA_pipeline.wdl [hunk 2]
```diff
@@ -75,30 +81,38 @@ workflow RNA_pipeline {
...
-      fusion_inspector="validate"
-  }
+#  call hg38_star_fusion.star_fusion_hg38_wf as StarFusion {
+#    input:
+#      left_fq=samtofastq.fastq1,
+#      right_fq=samtofastq.fastq2,
+#      sample_id=sample_id,
+#      genome_plug_n_play_tar_gz=genome_plug_n_play_tar_gz,
+#      fusion_inspector="validate"
+#  }
```
Comment (5im1z) [inline RNA_pipeline/RNA_pipeline.wdl:108]: why are these commented out?

File: RNA_pipeline/dalmatian_update_strandness.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import dalmatian as dm
+import numpy as np
+import pandas as pd
+
+
+def main():
+    wm = dm.WorkspaceManager("broad-firecloud-ccle/DEV_DepMap_hg38_RNAseq")
+    attributes_df = wm.get_samples()
+    strandness_df = pd.read_csv("/home/ubuntu/depmap_omics/ccle_tasks/rna_strandness_24q2.csv", index_col=0)
```
Comment (5im1z) [inline RNA_pipeline/dalmatian_update_strandness.py:9]: again, I don't think the df and the workspace name should be hardcoded

File: RNA_pipeline/star_fusion_hg38.wdl [hunk 4]
```diff
@@ -1,6 +1,6 @@
 version 1.0
 
-import "https://raw.githubusercontent.com/STAR-Fusion/STAR-Fusion/Terra-1.10.1/WDL/star_fusion_workflow.wdl" as SFW
+import "star_fusion_workflow.wdl" as SFW
```
Comment (5im1z) [inline RNA_pipeline/star_fusion_hg38.wdl:3]: why?

**Review by 5im1z — COMMENTED** [review]
Not sure if I missed anything but looks like this PR has things that are not related to the strandedness correction. I don't know if they need to be merged in

**Comment by gulatide** [comment]
@qinqian This PR has 30 commits - can we split into smaller PRs next time so it is more digestible?

**Pull request closed without merging**